zerver/views/users.py

```diff
@@ -416,11 +416,21 @@ def get_members_backend(request: HttpRequest, user_profile: UserProfile, user_id
     if user_id is not None:
         target_user = access_user_by_id(user_profile, user_id, allow_deactivated=True,
                                         read_only=True)
+        key = "user"
+    else:
+        key = "members"
```

Comment:
I think the code is clearer with a bit of code duplication without this `key` variable.